appium/webdriver/extensions/flutter_integration/flutter_commands.py

```diff
@@ -0,0 +1,179 @@
+            opts = {'element': locator, 'timeout': time_out}
+        else:
+            opts = {'locator': locator.to_dict(), 'timeout': time_out}
+        
+        self.execute_flutter_command('waitForAbsent', opts)
+
+    # flutter action commands
+    
+    def perform_double_click(self, element: WebElement, offset: Optional[Tuple[int, int]] = None) -> None:
+        """
+        Performs a double-click on the given element, with an optional offset.
+
+        Args:
+            element (WebElement): The element to double-click on. This parameter is required.
+            offset (Optional[Tuple[int, int]]): The x and y offsets from the element to click at. If not specified, the click is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset is not None:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('doubleClick', opts)
+    
+    def perform_long_press(self, element: WebElement, offset: Optional[Tuple[int, int]] = None) -> None:
+        """
+        Performs a long press on the given element, with an optional offset.
+
+        Args:
+            element (WebElement): The element to perform the long press on. This parameter is required.
+            offset (Optional[Tuple[int, int]]): The x and y offsets from the element to perform the long press at. If not specified, the long press is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset is not None:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('longPress', opts)
+            
+    def perform_drag_and_drop(self, source: WebElement, target: WebElement) -> None:
+        """
+        Performs a drag-and-drop operation from a source element to a target element.
+
+        Args:
+            source (WebElement): The element to drag from.
+            target (WebElement): The element to drop onto.
+            
+        Returns:
+            None:
+        """
+        self.execute_flutter_command('dragAndDrop', {'source': source, 'target': target})
+        
+    def scroll_till_visible(self, scroll_to: FlutterFinder, scroll_direction: Optional[ScrollDirection] = ScrollDirection.DOWN, **opts: Any) -> WebElement:
+        """
+        Scrolls until the specified element becomes visible.
+
+        Args:
+            scroll_to (FlutterFinder): The Flutter element to scroll to.
+            scroll_direction (Optional[ScrollDirection]): The direction to scroll up or down. Defaults to `ScrollDirection.DOWN`.
+                
+        KeywordArgs:
+                scrollView (str): The view of the scroll.
+                delta (int): delta for the scroll
+                maxScrolls (int): Max times to scroll
+                settleBetweenScrollsTimeout (int): settle timeout
```

Comment:
Please mention explicitly the time unit used there (I assume milliseconds) and default values if not provided

Reply:
fixed and added the default values

Reply:
hm, I still cannot find neither for KeywordArgs

Reply:
I have pushed the changes.